src/js/tabs/firmware_flasher.js

```diff
@@ -1241,22 +1241,20 @@ TABS.firmware_flasher.cleanup = function (callback) {
     if (callback) callback();
 };
 
-TABS.firmware_flasher.enableFlashing = function (enabled) {
-    var self = this;
-
+firmware_flasher.enableFlashing = function (enabled) {
     if (enabled) {
         $('a.flash_firmware').removeClass('disabled');
     } else {
         $('a.flash_firmware').addClass('disabled');
     }
 };
 
-TABS.firmware_flasher.FLASH_MESSAGE_TYPES = {NEUTRAL : 'NEUTRAL',
+firmware_flasher.FLASH_MESSAGE_TYPES = {NEUTRAL : 'NEUTRAL',
                                              VALID   : 'VALID',
                                              INVALID : 'INVALID',
                                              ACTION  : 'ACTION'};
 
```

Comment:
The alignment of the elements has been broken. We can keep them aligned o simply change it to something less fancy but less dependent of changes:
```javascript
firmware_flasher.FLASH_MESSAGE_TYPES = {
    NEUTRAL : 'NEUTRAL', 
    VALID : 'VALID',
    INVALID : 'INVALID',
    ACTION : 'ACTION',
};
```